src/main/java/justin/Justin.java

```diff
@@ -0,0 +1,159 @@
+package justin;
+
+import java.io.File;
+import java.util.Scanner;
+
+public class Justin {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+
+    public Justin(String filePath) {
+
+        ui = new Ui();
+        this.storage = new Storage(filePath);
+        tasks = storage.loadFile(filePath);
+
+    }
+
+    public void run() {
+
+        /*
+         * justin.Justin is a chatbot that help users plan and organise tasks
+         * justin.Justin stands for JUSt a TImetable(New) : JUSTIN
+         *
+         * justin.Justin is able to create and manage new tasks
+         *   1) Mark tasks off as done
+         *   2) Set tasks as To Do's - <keyword> <name>
+         *   3) Set tasks as justin.Deadline's - keyword  : <keyword> <name> /by <day>
+         *   4) Set tasks as justin.Event's - keyword : <keyword> <name> /at <day time>
+         *   5) Supports deletion of completed tasks with command delete <int>
+         *
+         * justin.Justin also supports viewing the entire tasks list with the command list - keyword : <keyword>
+         * To end off the session user can input bye to terminate program - keyword : <keyword>
+         *
+         * @author Goh Wei Kiat aka github : mrweikiat
+         * @version CS2103T AY20/21 Semester 2, Individual Project 'IP'
+         */
+
+
+        Scanner sc = new Scanner(System.in);
+
+        // Starting line for UI
+        ui.showWelcomeMessage();
+        ui.showHelpMessage();
+
+        // Condition for Duke to stop
+        boolean terminate = false;
+
+        //Duke will keep repeating until command given "Bye"
+        while (!terminate) {
+
+            String text = sc.nextLine();
+
+            Parser pr = new Parser(text);
+
+            try {
+                String command = pr.checkCommand();
+
+                switch (command) {
+
+                    case "BYE":
+
+                        storage.saveFile(tasks, storage.getFilePath());
+                        terminate = true; // terminates Duke
+                        ui.showEndMessage();
+                        break;
+
+                    case "LIST":
+
+                        ui.showListMessage();
+                        ui.printList(tasks);
+                        break;
+
+                    case "DONE":
+
+                        String num = text.substring(5); // take out the int value of the task to be completed
```

Comment:
Would it be clearer if a named constant or method was used to extract the task number instead? (instead of using a "magic" number)